frontend/src/components/SignInform.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from 'react'
+import { useHistory } from 'react-router-dom'
+
+// lib
+import { Article } from '../lib/FormStyle'
+import { Form } from '../lib/FormStyle'
+import { Input } from '../lib/FormStyle'
+
+export const SignInform = () => {
+  
+const [signInUser, setsignInUser] = useState({
+  email: '',
+  password: ''
+})
+const history = useHistory()
+const [error, setError] = useState('')
+
+const handleSubmit = event => {
+event.preventDefault()
+
+
+fetch("https://project-auth-ebba-elin.herokuapp.com/sessions",
+  {
+    method: 'POST',
+    headers: {'Content-Type': 'application/json'},
+    body: JSON.stringify(signInUser)
+  }
+).then(res => {
+  if (!res.ok) {
+    throw new Error('Unable to sign in, please try again')
+  }
+  res.json().then(data => {
+    if (data.notFound !== true) {
```

Comment:
This line gets a little confusing to read with the double negatives. Is it possible to rewrite it to
```
if (data.found) {
  ...
}
```
?